packages/http-client-python/package.json

```diff
@@ -80,7 +80,7 @@
     "typescript": "~5.6.2",
     "typescript-eslint": "^8.5.0",
     "@azure-tools/typespec-azure-core": "~0.47.0",
-    "@azure-tools/typespec-client-generator-core": "~0.47.1",
+    "@azure-tools/typespec-client-generator-core": "~0.47.4",
```

Comment:
https://github.com/Azure/autorest.python/pull/2898/files#r1827315298 TCGC 47.4 introduces some models which are useless for Python. Is it expected?

Reply:
added filtering logic

Reply:
According to https://github.com/Azure/autorest.python/pull/2900, current PR will cause no diff in generated SDK.